packages/eslint-config-airbnb/test/test-base.js

```diff
@@ -1,13 +1,13 @@
 import test from 'tape';
 import base from '../base';
 
-test('base: does not reference react', t => {
-  t.plan(2);
+test('base: does not reference react', tst => {
```

Comment:
tbh this type of change to me is an indicator that perhaps we **don't** wan't `id-length` in the first place - `t` is the conventional argument name here for tape, and using an abbreviated "test" seems hacky.